BirthdayGreetingsTest/api.js

```diff
@@ -12,50 +12,45 @@ const app = express();
 const port = 3000;
 app.use(express.json());
 
-app.get("/birthdays/:date", async (req, res) => {
-  // Get all friend objects that have a birthday on the specified date
-  const date = req.params.date;
-  retrieveBirthdaysFromDB(date)
-    .then((friends) => {
-      res.send(friends);
-    })
-    .catch((err) => {
-      res.status(500).send(err);
-    });
+app.get("/birthdays", async (req, res) => {
```

Comment:
Very minor thing but when we hit this endpoint what are we getting? Friends? Contacts? The url describes the resource the client is interacting with. 

https://www.partech.nl/nl/publicaties/2020/07/9-trending-best-practices-for-rest-api-development#

Reply:
@RichardWatkins1 Would something like `app.get("/birthday-friends"` be okay? I'm finding these rest api naming conventions a little confusing.